Dockerfile

```diff
@@ -16,8 +16,9 @@ WORKDIR /usr/local/bin/hypertrace
 
 COPY --from=build-stage /go/src/github.com/hypertrace/collector/collector .
 COPY default-config.yml /etc/opt/hypertrace/config.yml
+COPY default-ht-config.yml /etc/opt/hypertrace/ht-config.yml
 
 EXPOSE 9411
 
 ENTRYPOINT ["/usr/local/bin/hypertrace/collector"]
-CMD ["--config", "/etc/opt/hypertrace/config.yml"]
+CMD ["--config", "/etc/opt/hypertrace/ht-config.yml"]
```

Comment:
I think you want to continue with,
```
CMD ["--config", "/etc/opt/hypertrace/config.yml"]
```

Reply:
https://github.com/hypertrace/hypertrace-collector/pull/45#discussion_r587246227 I was going with @jcchavezs 's comment here.

Reply:
Then, do we need one more config `default-config.yml` file? As we are going with ENV, let's just have one file then.

Reply:
`default-config.yml` has prometheus and other things which we don't use in docker-compose so it basically exits at that point. 

I will keep them separate and let people use default-configs and tweak them according to need

Reply:
I agree if we use env vars we don't need to have two separated files for default config.